orion/sites/admin/src/achieve/achieve.js

```diff
@@ -12,16 +13,19 @@ export class AchievePage extends React.Component {
 	    };
  	}
 
+	componentDidMount() {
+      API.get("api/achievements/v1/all")
+        .then(res => {
+          const achievements = res.data;
+          this.setState({ list: achievements });
+        });
+    }
+
 	render() {
-		var numAchievements = 5;
-		var fakeAchieve = {
-			year: 2020,
-			message: "Hi"
-		};
-		var fakeAchieve2 = {
-			year: 2019,
-			message: "Hello"
-		};
+		const achieve = this.state.list.map((achieve,index) => {
+			return <AchieveRow key ={index} achieve = {achieve}/>
```

Comment:
nit: Just be careful of spacing here. Should be:
```
<AchieveRow key={index} achieve={achieve}/>
```
No spaces between the equal signs.